### PR TITLE
(#4223) Initial Cancer Types Landing Page YAML

### LINKED ISSUE #4223: Task: Create Initial Cancer Type Landing Page YAML
### Tasks

- [ ] Create that JS/CSS entry point in the front-end code
- [ ] Add the Drupal library
- [ ] Make the YAML content that references that Drupal library
  - Home and Landing Page Style: NCIDS with Title
  - Site Section: Make this the new Cancer Types. Make current Cancer Types into Cancer Types /cgdpl
  - Page Title: Cancer Types
  - Browser Title: A to Z List of Cancer Types
  - Card Types: Cancer Types
  - Meta Description: A to Z list of cancer types, with links to disease-specific and general information about treatment, supportive care, screening, prevention, clinical trials, and other topics.
  - Contents: One Column Container
    - Main Contents: Raw HTML Content (add basic structure if able. Otherwise, leave for the future tickets that will build out the rest of the work in R2.4.
  - Enable using CSS/JS on the right column once added per #4236 
  - For all other fields, if not indicated, leave blank.

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
Closes #4223

*This PR is currently rebased off of #4263*

PR Notes:

- Created a new library called `addl-types-landing-page` which is currently empty and is importing USWDS
- We will need to add the imports for the accordion and combo box once we bump the DS package
- This branch is currently rebased off of #4263

### CODE REVIEW

**Comment by andyvanavery31** [comment]
Looks good to me!

**Review by adrianacastaneda — APPROVED** [review]
Looks good to me as well!

**Comment by blilianyu** [comment]
Looks good to me as well!

**Review by bennettcc — APPROVED** [review]

**Comment by bennettcc** [comment]
Approving on behalf of product - Andy, Adriana, Lilian approved on 2/28

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/webpack.entries.js [hunk 1]
```diff
@@ -47,4 +47,5 @@ module.exports = {
 		'src/entrypoints/app-module/app-module.ts'
 	),
 	'addl-lib-test': path.resolve(__dirname, 'src/entrypoints/addl-lib-test/addl-lib-test.ts'),
+	'addl-types-landing-page': path.resolve(__dirname, 'src/entrypoints/addl-types-landing-page/addl-types-landing-page.ts'),
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/webpack.entries.js:50]: Put a comment in as to what this is for. The name actually confused me for a moment. Is this "Additional Types" for "Landing Page content type?". This probably should be `addl-cancer-types-landing-page` or something to make it REAL clear what it is for.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
I have one small thing, otherwise it looks good.

**Pull request closed without merging**